aerospike/assets/configuration/spec.yaml

```
@@ -487,5 +362,129 @@ files:
           - dc_ship_destination_error
           - dc_ship_source_error
           - dc_ship_success
+  - template: instances
+    options:
+    - template: instances/openmetrics
+      overrides:
+        openmetrics_endpoint.enabled: true
```

```suggestion
        openmetrics_endpoint.enabled: true
```
Isn't that the default?

---

I think in this case, because we are overriding the the `required: true` setting to allow users to still use the legacy check, it is needed in order for it to also be uncommented in the yaml. I followed the model for [Kong](https://github.com/DataDog/integrations-core/blob/master/kong/assets/configuration/spec.yaml#L12-L13) here.